fedb/api-server.md

```diff
@@ -0,0 +1,157 @@
+- Start Date: 2021-04-23
+- Target Major Version: (2.2.0)
+- Reference Issues: 
+- Implementation PR: 
+
+# Add API Server
+
+# Summary
+
+HTTP interface is one of the most friendly interface and many developer like using http interface to develop their program. HTTP API also help us to show demo simply. So we want to support uing http api to access FEDB.
+
+# Detailed design
+
+
+## Interface design
+HTTP URL: http://ip:port/sql
+
+HTTP Method: POST
+
+### Put
+request
+```
+{
+    "method": "put",
```

Comment:
Current description is a bit hard to determine the interface, which is expected?

1. `PUT /sql/table/{table_name}/db/{db_name}`
2. `PUT /sql?table={table_name}&db={db_name}`
3. or put all the params into request body?